mlx/ops.cpp

```diff
@@ -696,6 +696,54 @@ split(const array& a, int num_splits, StreamOrDevice s /* = {} */) {
   return split(a, num_splits, 0, to_stream(s));
 }
 
+std::vector<array> meshgrid(
+    std::vector<array>& arrays,
+    const bool copy /* = true */,
```

Comment:
This doesn't really make sense in MLX since operations are really always returning a new array. For instance you can't have a "view" of one array that is then updated and the view is updated as well (which is kind of the point for this argument).